apps/vela-ext/tests/content.test.ts

```
@@ -0,0 +1,50 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
```

ESLint

[error] 'vi' is defined but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)